tests/py/dynamo/lowering/test_decompositions.py

```diff
@@ -1656,6 +1662,276 @@ def forward(self, input, weight, bias, running_mean=None, running_var=None):
+                    False,
+                    scale=scale,
+                )[0]
+
+        # Operations expected to be removed in the traced graph after decompositions
+        unexpected_ops = {torch.ops.aten._scaled_dot_product_flash_attention.default}
+
+        inputs = [
+            torch.rand(2, 4, 8, 16, dtype=torch.half, device="cuda"),
+            torch.rand(2, 4, 8, 16, dtype=torch.half, device="cuda"),
+            torch.rand(2, 4, 8, 16, dtype=torch.half, device="cuda"),
+        ]
+
+        fx_graph = torch.fx.symbolic_trace(TestModule())
+        unexpected_ops_seen, _ = lower_graph_testing(
+            fx_graph, inputs, unexpected_ops=unexpected_ops, min_block_size=1
+        )
+
+        self.assertEqual(
+            len(unexpected_ops_seen),
+            0,
+            f"The following unexpected ops were encountered: {unexpected_ops_seen}",
+        )
+
+        torch._dynamo.reset()
+
+        # Validate that the results between Torch and Torch-TRT are similar
+        optimized_model = torch_tensorrt.compile(
+            fx_graph,
+            "dynamo",
+            inputs,
+            enabled_precisions={torch.half},
+            min_block_size=1,
+        )
+        optimized_model_results = optimized_model(*inputs).detach().cpu()
+        torch_model_results = fx_graph(*inputs).detach().cpu()
+
+        max_diff = float(
+            torch.max(torch.abs(optimized_model_results - torch_model_results))
+        )
+        self.assertAlmostEqual(
+            max_diff,
+            0,
+            DECIMALS_OF_AGREEMENT - 1,
+            "Scaled_dot_product_flash_attention TRT outputs don't match with the original model.",
+        )
+
+    @parameterized.expand(
+        [
+            (True, False, None),
+            (False, True, 0.123),
+        ]
+    )
+    def test_lowering_scaled_dot_product_efficient_attention(
+        self, attn, is_causal, scale
+    ):
+        class TestModule(torch.nn.Module):
+            def forward(self, query, key, value, attn_bias=None):
+                return torch.ops.aten._scaled_dot_product_efficient_attention.default(
+                    query,
+                    key,
+                    value,
+                    attn_bias,
+                    False,
+                    0.0,
+                    is_causal,
+                    scale=scale,
+                )[0]
+
+        # Operations expected to be removed in the traced graph after decompositions
+        unexpected_ops = {
+            torch.ops.aten._scaled_dot_product_efficient_attention.default
+        }
+
+        inputs = [
+            torch.rand(2, 4, 8, 16, dtype=torch.half, device="cuda"),
+            torch.rand(2, 4, 8, 16, dtype=torch.half, device="cuda"),
+            torch.rand(2, 4, 8, 16, dtype=torch.half, device="cuda"),
+        ]
+        if attn:
+            inputs += [torch.rand(2, 4, 8, 8, dtype=torch.half, device="cuda")]
+
+        fx_graph = torch.fx.symbolic_trace(TestModule())
```

Comment:
Can you try to use torch.export.export() to trace the graph ? eg: We are using this tracer for our converters